### DISCUSSION TITLE
403 session token has been expired

### DISCUSSION THREAD

**Discussion by abubakar0101** [Kyon147/laravel-shopify#85] [General]
when I uploading multiple file Images throw axios request then it take time and request genrate error (403 session token has been expired)

**Comment by Kyon147** [comment]
@abubakar0101 this sounds like your upload is taking longer than the 60 second token expiration. 

You will need to change your logic, so that you get the token before each upload or chunk up your upload so that it completes quicker for each file or batch of files.

**Comment by ghanshyam1994** [comment]
public const LEEWAY_SECONDS = 10;

issue is here ... :(

namespace Osiset\ShopifyApp\Objects\Values;

final class SessionToken implements SessionTokenValue
{

}

In This class I don't know how to resolve it.

it make expired session only in 10 sec.

**Reply by ghanshyam1994** [reply]
I want to crate override & create any configurations but nothing work could you please , let me know how to resolve this
